### PR TITLE
Support for topics settings deletion

### PR DESCRIPTION
**Pull request by fpacifici**
Same as https://github.com/segmentio/topicctl/pull/198
This is a PR in our fork so we can  merge it in master here and start working on that while the PR on upstream gets reviewed.

### CODE REVIEW

File: cmd/topicctl/subcmd/apply.go [hunk 1]
```diff
@@ -107,6 +108,12 @@ func init() {
 		false,
 		"Don't return error when topic's config specifies fewer partitions than it currently has",
 	)
+	applyCmd.Flags().BoolVar(
+		&applyConfig.destructive,
+		"destructive",
+		false,
+		"Deletes topic settings from the broker if the setting is set on the broker but not in config",
```
Comment (victoria-yining-huang) [inline cmd/topicctl/subcmd/apply.go:115]:
```suggestion
		"Deletes topic settings from the broker if the settings are present on the broker but not in the config"
```

File: cmd/topicctl/subcmd/apply.go [hunk 2]
```diff
@@ -259,7 +266,7 @@ func applyTopic(
 			RetentionDropStepDuration:  applyConfig.retentionDropStepDuration,
 			SkipConfirm:                applyConfig.skipConfirm,
 			IgnoreFewerPartitionsError: applyConfig.ignoreFewerPartitionsError,
-			SleepLoopDuration:          applyConfig.sleepLoopDuration,
```
Comment (victoria-yining-huang) [inline cmd/topicctl/subcmd/apply.go:262]: why is it deleted? intentional?

Comment (fpacifici) [reply cmd/topicctl/subcmd/apply.go:262]: ooops, I need to fix the other one then

Comment (fpacifici) [reply cmd/topicctl/subcmd/apply.go:262]: fixed

File: cmd/topicctl/subcmd/apply.go (hunk 1 above)
Comment (fpacifici) [reply cmd/topicctl/subcmd/apply.go]: Fixed

File: cmd/topicctl/subcmd/apply.go (hunk 2 above)
Comment (fpacifici) [reply cmd/topicctl/subcmd/apply.go:262]: fixed

**Review by bmckerry — APPROVED** [review]

**Pull request merged**